### ISSUE TITLE
Linkset ensembl for OpenPhacts

### ISSUE THREAD

**Issue by DeniseSl22** [bridgedb/website#8]
See this page: https://www.bridgedb.org/mapping-databases/linksets/

No clear description of this linkset, nor which data was used by OpenPhacts, nor how other people can use it. Anyone beside Jonathan that knows about the meaning of this page? @egonw @Chris-Evelo ?

**Comment by Chris-Evelo** [comment]
Kind of, yes. Basically, these are the linksets used by the semantic web version of BridgeDb, also known as the IMS. They contain one-on-one mappings between difference databases, derived from columns in the corresponding Derby database files.  They should all come with a VOID header that describes how and when they were created.

But you probably are best helped with existing descriptions. This "should" be in the Open PHACTS architecture paper. Also, Freddie described a use case for the IMS and IRS (in container form I think) for the recently (pre-)published gene to variant paper. It would be good to add a basic description, like what I wrote above, and provide links to the papers here. I think we should have a link to the front page of BridgeDb to a place where we describe the IMS, the IRS (identifier resolution server, which basically extends a query using IMS mappings) and the linksets themselves. This page should be linked from there and be linked back to it. While it should also links to and from the Derby database descriptive page.

**Comment by Chris-Evelo** [comment]
BTW I am almost sure we also dis metabolite ID linksets. @egonw ?

**Comment by egonw** [comment]
A short update: @nunogit has been working on generating linksets on Jenkins (for gene/protein).

**Comment by egonw** [comment]
@Chris-Evelo, we only did that as a temporary workaround, pending release of the chemistry resolution system (i.e. chemical lenses).

**Comment by Chris-Evelo** [comment]
I am confused now. The CRS is meant to allow us to resolve (sub)structures to identifiers for compounds that match these (sub)structures. The lenses are more a BridgeDb thing, switching between linksets (e.g. for racemates and stereochemically specific) I thought, and can thus be used for other things like gene products too.. Am I wrong?

**Comment by DeniseSl22** [comment]
@egonw @Chris-Evelo : I'm not sure what the status is of the IMS/IRS/CRS; I wanted to check the LinkSets mentioned above on jenkins (I thought that job was not running properly), but Jenkins is down now (@nunogit is working on it). And I believe the IMS should be reimplemented/revised for OpenPhacts right? Let me know if there are any updates on this matter.

**Comment by Chris-Evelo** [comment]
Yes, we will need to review the status of the IMS next week at the "federated Open PHACTS meeting anyway. I indeed heard that it is currently not working. I am not sure what the problem is or whether it was part of the OpenRiskNet or Freddie's container solutions. I know Alasdair wanted to have his new Research Assistant look at it. But that one left recently. We will also look at the IRS and see whether can get that to OLS I expect. But that is not really BridgeDb related (Or did i miss something?). The CRS needs to be redone and Egon got funding from COST CHARME to organize a meeting to create that based on the CDK. That should be planned soon since CHARME is in its final year.
